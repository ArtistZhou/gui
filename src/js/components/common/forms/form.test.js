import React from 'react';
import renderer from 'react-test-renderer';
import Form from './form';

describe('Form Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Form showButtons={true} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
