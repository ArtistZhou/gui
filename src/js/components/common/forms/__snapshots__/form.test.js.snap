// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<form
  className=""
>
  <div
    className="float-right"
    style={
      Object {
        "marginTop": "32px",
      }
    }
  >
    <button
      className="MuiButtonBase-root-27 MuiButtonBase-disabled-28 MuiButton-root-1 MuiButton-contained-12 MuiButton-raised-15 MuiButton-disabled-21"
      disabled={true}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="-1"
      type="button"
    >
      <span
        className="MuiButton-label-2"
      />
    </button>
  </div>
</form>
`;
